packages/web/src/store/tipping/sagas.ts

```diff
@@ -78,16 +81,9 @@ function* sendTipAsync() {
       yield call(walletClient.transferTokensFromEthToSol)
     }
 
-    try {
-      yield call(() =>
-        walletClient.sendWAudioTokens(recipientWallet, weiBNAmount)
-      )
-    } catch (e) {
-      const error = (e as Error).message
-      console.error(`Send tip failed: ${error}`)
-      yield put(sendTipFailed({ error }))
-      return
-    }
+    yield call(() =>
+      walletClient.sendWAudioTokens(recipientWallet, weiBNAmount)
```

Comment:
i think you can use call with the method syntax: https://redux-saga.js.org/docs/api/#callcontext-fn-args